src/App.css

```diff
@@ -16,15 +16,28 @@ body {
 }
 
 .View--Login {
+  height: 100vh;
   background-image: linear-gradient(to bottom, #fff 400px, transparent 100%),
-    url(https://portal.kidsfirstdrc.org/static/media/background-science.68317e4e.jpg);
+    url(./assets/background-science.jpg);
+    .center.column{
+      min-width: 300px !important;
+    }
 }
 
-.Card--Login {
-  @extend .Card;
-  width: 620px;
+.ui.button.button--login{
+    margin-bottom: 10px;
```

Comment:
We can use Semantic's spacing to accomplish the same thing by setting `<Segment padded />`, this way we can reproduce the login component between apps without copy+pasting css and it will insure we have consistent levels of spacing when using Segments.